sdk/ai/ai-inference-rest/README.md

```diff
@@ -404,6 +404,48 @@ const response = await client.path("/chat/completions").post({
 console.log(`Chatbot: ${response.choices[0].message?.content}`);
 ```
 
+### Text Embeddings example
+
+This example demonstrates how to get text embeddings with Entra ID authentication. 
+
+```javascript
+import ModelClient, { isUnexpected } from "@azure-rest/ai-inference";
+import { DefaultAzureCredential } from "@azure/identity";
+
+const endpoint = "<your_model_endpoint>";
+const credential = new DefaultAzureCredential();
+
+async function main(){
+  const client = ModelClient(endpoint, credential);
+  const response = await client.path("/embeddings").post({
+    body: {
+      input: ["first phrase", "second phrase", "third phrase"]
+    }
+  });
+
+  if (isUnexpected(response)) {
+    throw response.body.error;
```

Comment:
`createRestError` from `@azure-rest/core-client` should be called here to build proper Azure error object.

Reply:
actually never mind, resolving.